dto/src/main/java/ca/on/oicr/gsi/runscanner/dto/dragen/DragenAnalysis.java

```diff
@@ -0,0 +1,28 @@
+package ca.on.oicr.gsi.runscanner.dto.dragen;
+
+import ca.on.oicr.gsi.runscanner.dto.Analysis;
+import ca.on.oicr.gsi.runscanner.dto.dragen.samplesheet.Samplesheet;
+import java.util.HashMap;
+import java.util.Map;
+
+// Represents an entire attempt at a DRAGEN Pipeline
+public class DragenAnalysis implements Analysis {
+  Samplesheet samplesheet;
```

Comment:
The analysis attempt has a samplesheet and BCLConvert has a different samplesheet?